pulser-simulation/pulser_simulation/qutip_backend.py

```diff
@@ -16,18 +16,28 @@
 
 from typing import Any
 
-from pulser import Sequence
-from pulser.backend.abc import Backend
-from pulser.backend.config import EmulatorConfig
+import numpy as np
+import qutip
+
+import pulser
+from pulser.backend.abc import Backend, EmulatorBackend
+from pulser.backend.config import EmulationConfig, EmulatorConfig
+from pulser.backend.default_observables import StateResult
+from pulser.backend.results import Results
 from pulser.noise_model import NoiseModel
+from pulser_simulation.qutip_config import QutipConfig
+from pulser_simulation.qutip_op import QutipOperator
+from pulser_simulation.qutip_state import QutipState
 from pulser_simulation.simconfig import SimConfig
-from pulser_simulation.simresults import SimulationResults
+from pulser_simulation.simresults import CoherentResults, SimulationResults
 from pulser_simulation.simulation import QutipEmulator
 
 
 class QutipBackend(Backend):
     """A backend for emulating the sequences using qutip.
 
+    Warning: soon to be deprecated, please use QutipBackendV2.
```

Comment:
nit: Formatting so it renders well in the docs
```suggestion
    Warning: 
        Soon to be deprecated, please use ``QutipBackendV2``.
```

Reply:
done! thanks!